In [14]:
a = 12_500 * 0.25
a

3125.0

\begin{equation}
500 = (n*0.25) * 0.16
\end{equation}


\begin{equation}
\frac{500}{0.16} * \frac{1}{0.25} = n
\end{equation}
 
 
\begin{equation}
3125 * \frac{1}{0.25} = n
\end{equation}
 
 
\begin{equation}
12500 = n
\end{equation}